src/main/java/com/artipie/VertxMain.java

```diff
@@ -115,7 +115,7 @@ public int start(final int apiport) throws IOException {
             auth -> vertx.deployVerticle(
                 new RestApi(
                     caches, settings.repoConfigsStorage(), settings.layout().toString(),
-                    apiport, settings.credentialsKey(), auth, settings.keyStore()
+                    apiport, settings.credentialsKey(), auth, settings.keyStore(), settings
```

Comment:
and this looks strange, as we first ask credentials key and key store from the settings and then pass the whole settings instance...

Reply:
@olenagerasimova Thanks, I have fixed, please review